packages/create-cloudflare/src/dialog.ts

```diff
@@ -0,0 +1,169 @@
+		gray(shapes.corners.br);
+
+	return [
+		prefix + topRow,
+		...lines.map((line) => {
+			let lineWidth = stripAnsi(line).length;
+
+			if (lineWidth > maxInnerWidth) {
+				// Truncate the label of the hyperlinks to avoid overflow
+				// Note: This assumes the label to have no ANSI code at the moment
+				line = line.replaceAll(linkRegex, (_, url, label) =>
+					hyperlink(
+						url,
+						label.slice(
+							0,
+							label.length - (lineWidth - maxInnerWidth) - ellipses.length,
+						) + ellipses,
+					),
+				);
+				lineWidth = stripAnsi(line).length;
+			}
+
+			if (lineWidth > maxInnerWidth) {
+				// If there is no link, truncate the text instead
+				// FIXME: This assumes the text to have no ANSI code at the moment
+				line =
+					line.slice(0, innerWidth - lineWidth - ellipses.length) + ellipses;
+				lineWidth = stripAnsi(line).length;
+			}
+
+			return (
+				prefix +
+				gray(border) +
+				padding +
+				line +
+				padding.repeat(innerWidth > lineWidth ? innerWidth - lineWidth : 0) +
+				padding +
+				gray(border)
+			);
+		}),
+		prefix + bottomRow,
+	].join("\n");
+}
+
+export function printWelcomeMessage(version: string) {
+	const dialog = createDialog([
+		`👋 Welcome to create-cloudflare v${version}!`,
+		`🧡 Let's get started.`,
+	]);
+
+	logRaw(dialog);
+}
+
+export const printSummary = async (ctx: C3Context) => {
+	// Prepare relevant information
+	const dashboardUrl = ctx.account
+		? `https://dash.cloudflare.com/?to=/:account/workers/services/view/${ctx.project.name}`
+		: null;
+	const relativePath = relative(ctx.originalCWD, ctx.project.path);
+	const cdCommand = relativePath ? `cd ${relativePath}` : null;
+	const { npm } = detectPackageManager();
+	const devServerCommand = quoteShellArgs([
+		npm,
+		"run",
+		ctx.template.devScript ?? "start",
+	]);
+	const deployCommand = quoteShellArgs([
+		npm,
+		"run",
+		ctx.template.deployScript ?? "deploy",
+	]);
+	const documentationUrl = `https://developers.cloudflare.com/${ctx.template.platform}`;
+	const discordUrl = `https://discord.cloudflare.com`;
+
+	// Prepare the dialog
+	const lines = [
+		`🎉 ${bgGreen(" SUCCESS ")} Application ${ctx.deployment.url ? "deployed" : "created"} successfully!`,
+		``,
+	];
+
+	if (ctx.deployment.url && dashboardUrl) {
+		lines.push(
+			`🔍 View Project`,
+			`   ${gray("Visit:")} ${blue.underline(hyperlink(ctx.deployment.url))}`,
+			`   ${gray("Dash:")} ${blue.underline(hyperlink(dashboardUrl))}`,
+			``,
+		);
+	}
+
+	lines.push(
+		`💻 Continue Developing`,
+		...(cdCommand
+			? [`   ${gray("Change directories:")} ${blue(cdCommand)}`]
+			: []),
+		`   ${gray("Start dev server:")} ${blue(devServerCommand)}`,
+		`   ${gray(ctx.deployment.url ? `Deploy again:` : "Deploy:")} ${blue(deployCommand)}`,
+		``,
+		`📖 Explore Documentation`,
+		`   ${blue.underline(hyperlink(documentationUrl))}`,
+		``,
+		`💬 Join our Community`,
+		`   ${blue.underline(hyperlink(discordUrl))}`,
+		``,
```

Comment:
I think we should have equal top and bottom margins in this dialog, it feels a little lopsided having the extra newline at the bottom:
![image](https://github.com/user-attachments/assets/c279bde1-3cd5-44f8-a801-d643b384a6ea)

Reply:
Also, I like the typography and icon changes here, but having a dialog in the summary feels a little weird since it's breaking from the convention we're using everywhere else of using `├` for headers and `│` for body text.

Reply:
Thanks for pointing this out! We have moved the dialog out of section 3. I have also updated the screenshots to reflect the changes.